src/data-table/data-table.tsx

```diff
@@ -256,18 +277,37 @@ export const DataTable = ({
         );
     };
 
+    const renderSelectionBar = () => {
+        return (
+            <SelectionBar>
+                <Text.H5 weight="semibold">{`${selectedIds.length} items selected`}</Text.H5>
+                <ClearSelectionAction onClick={onClearSelectionClick}>
+                    Clear Selection
+                </ClearSelectionAction>
```

Comment:
This should be a button as it's an interactive element. I think the closest DS component is `<Button.Small styleType="link">`